Respriting Medical

CorruptOmega opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Medical has been resprited.
Chemist now has roundstart hats (if they want them)
Added some new clothing that didn't exist, some new suits that didn't exist, shenanigans.

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
The spritelord willed it, so shall it be.

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
<img width="859" height="622" alt="image" src="https://github.com/user-attachments/assets/e2f23593-78cc-4d30-a00d-7bf5075cacfd" />
<img width="797" height="292" alt="image" src="https://github.com/user-attachments/assets/5d08bb9d-3361-4565-b509-f81fb7992ab3" />

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
:cl: CorruptOmega
- tweak: Medical department resprited
- add: New Medical EVA suit
- add: Chemist hat choice
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->

princess-gurchi commented on Resources/Prototypes/Recipes/Lathes/Packs/clothing.yml:
```diff
@@ -72,6 +72,9 @@
   - ClothingHeadHatParamedicsoft
+  - ClothingHeadHatBeretMedical #Far Horizons
+  - ClothingHeadHatBeretChemist #Far Horizons
+  - ClothingHeadHatBeretVirologist #Far Horizons
```
These are supposed to be `latheRecipe` entities, not items directly. Please add latheRecipes for these 3

princess-gurchi commented:
I can't explain it, but chemist beret just doesn't display in loadout preview. Everything else works, you can see it as soon as you spawn in, and I don't see anything wrong with yamls, but it just doesn't show.

<img width="1018" height="467" alt="image" src="https://github.com/user-attachments/assets/4adacd7a-be57-4158-9596-715c0d425083" />

This is not a reason to prevent a merge, but please take a look and maybe you'll see something I didn't.

CorruptOmega commented:
@princess-gurchi Yeah I also noticed it not being there on the loadout screen. I'll see if I can find out why later (I'm about to head to work). Does the medical beret show in that loadout screen?

princess-gurchi commented:
yep, medical beret is showing and I wasn't able to find any difference between them 2

CorruptOmega commented:
Fixed the Beret not showing in loadout, added lathe recipes for the Chemist Beret, Virology Beret and jumpsuit/jumpskirt into the Uniform Printer. Added Medical Doctor being able to choose Virology uniforms in loadout.

<img width="1790" height="748" alt="image" src="https://github.com/user-attachments/assets/a4b5458b-d1d9-43ed-8b46-93f88c06c7de" />
<img width="217" height="358" alt="image" src="https://github.com/user-attachments/assets/4f747c52-0073-404f-9f80-165c47c60c4f" />
<img width="394" height="502" alt="image" src="https://github.com/user-attachments/assets/49f673da-5ee7-4c90-8acf-15e30b899ca4" />
<img width="403" height="497" alt="image" src="https://github.com/user-attachments/assets/1b730bff-9ad3-44b3-9215-0a906fe83513" />
<img width="391" height="407" alt="image" src="https://github.com/user-attachments/assets/96c46a68-e88d-4fc2-8d35-e7e6fa9ff658" />

CorruptOmega commented:
@princess-gurchi Ready for a re-review and checking to make sure I didn't miss anything.
